### DISCUSSION TITLE
Disable/Enable vps over SSH

### DISCUSSION THREAD

**Discussion by sbfemeri** [Ysurac/openmptcprouter#3253] [General]
Hi guys. I want to disable or enable the VPS connection via the shell. At the moment, I'm using the web GUI (System/OpenMOPCTProuter/Wizard -> disable server). My plan is to write a script that can disable/enable the connection to the VPS over SSH. I would like to deactivate MPTCP to get less ping when gaming. My plan is to use a detector on my HomeKit setup, and if my PlayStation is connected to my router, HomeKit sends an SSH command to the OpenMPTCProuter.

**Comment by Ysurac** [comment]
You can set OMR-ByPass to bypass VPN and use directly a connection for the PlayStation. It will be far easier.

**Reply by sbfemeri** [reply]
First of all, many thanks for the ingenious project Ysurac. 
It is possible to disable/enable the bypass via ssh? After the OpenMPCTCProuter I connected my old router with wifi 6 mesh. So I need a script to switch on/off this bypass.

**Reply by Ysurac** [reply]
Yes something like ```uci set omr-bypass.ruleyouwanttomodify.enabled=0 && uci commit omr-bypass && /etc/init.d/omr-bypass reload``` should do the job (enabled=1 to enable, ```uci show omr-bypass``` to show entries)
